x-pack/plugin/runtime-fields/qa/src/main/java/org/elasticsearch/xpack/runtimefields/test/CoreTestTranslater.java

```diff
@@ -79,18 +80,18 @@
         NumberType.DOUBLE.typeName(),
         KeywordFieldMapper.CONTENT_TYPE,
         IpFieldMapper.CONTENT_TYPE,
-        NumberType.LONG.typeName()
+        NumberType.LONG.typeName(),
+        GeoPointFieldMapper.CONTENT_TYPE
```

Comment:
this is triggering new failures, so I guess I am better off making this change separately.